satpy/etc/composites/fci.yaml

```diff
@@ -11,3 +26,17 @@ composites:
       - name: 'cloud_state'
     lut: [.nan, 0, 1, 1, 1, 1, 1, 1, 0, .nan]
     standard_name: binary_cloud_mask
+
+  true_color:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
```

Comment:
what is the advantage here to use the SelfSharpenedRGB compositor? Aren't the channels already all in the same resolution?

Reply:
Or are you already thinking of the HRFI case? Would the sharpening work automatically if the vis_06 is provided in a higher resolution?

Reply:
I'm thinking of the HR case.  I believe the sharpening should work if `vis_06` is provided at 500 metre.

Reply:
Nice!